server/src/services/itoService.ts

```diff
@@ -403,7 +403,7 @@ export default (router: ConnectRouter) => {
           createdAt: new Date().toISOString(),
           updatedAt: new Date().toISOString(),
           llm: create(LlmSettingsSchema, {
-            asrModel: 'distil-whisper-large-v3-en',
+            asrModel: 'whisper-large-v3',
```

Comment:
This change is unrelated to this pr but I changed it because forces everyone to use a worse model. We should default to the model that we prefer to use.